src/codec.js

```diff
@@ -69,6 +69,80 @@ Int.prototype.valueOf = function() {
 
 codec.Int = Int;
 
+/**
+ * We use this symbol as a means to identify if an array is actually a struct.
+ * We need to do this because changing Structs from an object to an array would
+ * be a major breaking change.
```

Comment:
I don't understand this comment. can you please elaborate?

Reply:
When the types dictionary is omitted, we attempt to determine the type for the user. Because structs can have empty/duplicate field names, we've always used an Array to represent a struct. Now that we accept it as a param we need a way to distinguish between an actual array and a struct (assuming the types object is null).

Reply:
So to bind a struct parameter, users can either pass in a `Spanner.struct(object)` or an Array?

Reply:
Any updates?

Reply:
ping @callmehiphop

Reply:
@vkedia thats correct